crates/router/src/connector/stripe.rs

```diff
@@ -1929,3 +1952,484 @@ impl services::ConnectorRedirectResponse for Stripe {
+        self.build_headers(req, connectors)
+    }
+
+    fn get_request_body(
+        &self,
+        req: &types::PayoutsRouterData<api::PoRecipient>,
+    ) -> CustomResult<Option<types::RequestBody>, errors::ConnectorError> {
+        let connector_req = stripe::StripeConnectRecipientCreateRequest::try_from(req)?;
+        let wise_req = types::RequestBody::log_and_get_request_body(
+            &connector_req,
+            utils::Encode::<stripe::StripeConnectRecipientCreateRequest>::url_encode,
+        )
+        .change_context(errors::ConnectorError::RequestEncodingFailed)?;
+        Ok(Some(wise_req))
+    }
+
+    fn build_request(
+        &self,
+        req: &types::PayoutsRouterData<api::PoRecipient>,
+        connectors: &settings::Connectors,
+    ) -> CustomResult<Option<services::Request>, errors::ConnectorError> {
+        let request = services::RequestBuilder::new()
+            .method(services::Method::Post)
+            .url(&types::PayoutRecipientType::get_url(self, req, connectors)?)
+            .attach_default_headers()
+            .headers(types::PayoutRecipientType::get_headers(
+                self, req, connectors,
+            )?)
+            .body(types::PayoutRecipientType::get_request_body(self, req)?)
+            .build();
+
+        Ok(Some(request))
+    }
+
+    #[instrument(skip_all)]
+    fn handle_response(
+        &self,
+        data: &types::PayoutsRouterData<api::PoRecipient>,
+        res: types::Response,
+    ) -> CustomResult<types::PayoutsRouterData<api::PoRecipient>, errors::ConnectorError> {
+        let response: stripe::StripeConnectRecipientCreateResponse = res
+            .response
+            .parse_struct("StripeConnectRecipientCreateResponse")
+            .change_context(errors::ConnectorError::ResponseDeserializationFailed)?;
+        types::RouterData::try_from(types::ResponseRouterData {
+            response,
+            data: data.clone(),
+            http_code: res.status_code,
+        })
+    }
+
+    fn get_error_response(
+        &self,
+        res: types::Response,
+    ) -> CustomResult<types::ErrorResponse, errors::ConnectorError> {
+        self.build_error_response(res)
+    }
+}
+
+#[cfg(feature = "payouts")]
+impl
+    services::ConnectorIntegration<
+        api::PoRecipientAccount,
+        types::PayoutsData,
+        types::PayoutsResponseData,
+    > for Stripe
+{
+    fn get_content_type(&self) -> &'static str {
+        self.common_get_content_type()
+    }
+
+    fn get_url(
+        &self,
+        req: &types::PayoutsRouterData<api::PoRecipientAccount>,
+        connectors: &settings::Connectors,
+    ) -> CustomResult<String, errors::ConnectorError> {
+        let connector_customer_id = req
+            .connector_customer
+            .clone()
+            .get_required_value("connector_customer")
+            .change_context(errors::ConnectorError::MissingRequiredField {
+                field_name: "connector_customer",
```

Comment:
If it is being used at multiple places, also can be used across multiple connectors, then move it to `connector/utils.rs`